src/image/index.ts

```diff
@@ -196,6 +196,9 @@ export const image = <
     schema: () => ({
       ...def,
       ...(fields && fieldsSchema(fields)),
+      ...(hotspot === true
+        ? { options: { ...def.options, hotspot: true } }
+        : {}),
```

Comment:
Two comments:

- The ternary is wrong. You don't want to return `{}`, you want to return `def.options`.
- Follow the same pattern you see in other types. Destructure `options` in the args, and then do this instead:
```diff
-      ...(hotspot === true
-        ? { options: { ...def.options, hotspot: true } }
-        : {}),
+      options: { ...options, hotspot },
```

Reply:
Sorry for the late reply, good catch on the ternary, should prolly cover that with a test. The issue with the change you're suggesting is that it'll pass through `hotspot: undefined`, when hotspot isn't enabled and thus `image › builds a sanity config` fails. Do you want me to correct the test, or do something like this:

```typescript
options: { ...options, ...( hotspot != null ? {hotspot} : {} ) }
```
?

Personally i prefer that the output matches the input, (ie. i don't get a `hotspot` key on `options` if undefined).

Reply:
My preference is that ternaries are generally better than `if` statements, but can often be avoided with sensible defaults and look cleaner that way. For example:

```diff
// Not this
-      ...(hotspot === true
-        ? { options: { ...def.options, hotspot: true } }
-        : {}),

// or this
-      options: { ...options, ...( hotspot != null ? {hotspot} : {} ) }

// but this:
+      options: { ...options, hotspot: Boolean(hotspot) },
```

Reply:
Ah naturally! Much better, will get that fixed 👍

Reply:
@saiichihashimoto I suppose you'd like me to change the expected value in [`image › builds a sanity config`](https://github.com/saiichihashimoto/sanity-typed-schema-builder/blob/master/src/image/index.spec.ts#L17) since it's now always present?